assets/js/event-providers/contact-form-7.js

```diff
@@ -15,6 +15,13 @@
  */
 
 global.document.addEventListener( 'wpcf7mailsent', ( event ) => {
+	// Prevent sending duplicate event with different names when Popup Maker is active.
+	// PUM also tracks submissions from CF7 as `submit_lead_form`, which is not throttled
+	// as key is going to differ due to the different event names.
+	if ( typeof PUM !== 'undefined' ) {
+		return;
+	}
+
```

Comment:
@zutigrm I dont think this is going to work. Let's remove this completely and leave it as is. The fact that we have two events triggered when CF is used in PUM is okay and we shouldn't prevent that because those event names are different. cc @mohitwp @wpdarren